recipes/ants/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -ex
+
+mkdir build
+cd build
+
+cmake -G Ninja \
+    ${CMAKE_ARGS} \
+    -DCMAKE_INSTALL_LIBDIR:STRING=lib \
+    -DCMAKE_INSTALL_PREFIX:STRING=${PREFIX} \
+    -DCMAKE_PREFIX_PATH:STRING=${PREFIX} \
+    -DBUILD_SHARED_LIBS:BOOL=ON \
+    -DANTS_SUPERBUILD:BOOL=OFF \
+    -DUSE_SYSTEM_ITK:BOOL=ON \
+    ${SRC_DIR}
+
+cmake --build . --config RelWithDebInfo --parallel ${CPU_COUNT} --target install
```

Comment:
Is there a `make check`  or `ctest` we can run here?

Reply:
Indeed, there is :+1: 

Thanks for pointing it out. I have updated the recipe to run the test suite after the build.